ArticleTemplates/articleTemplateContainerImmersive.html

```diff
@@ -33,7 +33,7 @@ <h1 class="article__headline headline selectable">
     </div>
 
     <div class="article__body" id="article-body">
-        <div class="from-content-api prose resizable selectable">
+        <div class="from-content-api prose resizable resizable_line_height selectable">
```

Comment:
Nit: check whether this should be with hyphens or not, most class names seem to use them, although `article__body` above doesn't...

Reply:
This should be with hyphens, BEM (http://getbem.com/introduction/) is the naming convention used in the CSS. `article__body` is actually a "block" `article` and "element" `body`, which are separated by double underscores.